tests/model_registry/rest_api/test_model_registry_secure_db.py

```diff
@@ -13,7 +13,7 @@
 LOGGER = get_logger(name=__name__)
 
 
-@pytest.mark.usefixtures("updated_dsc_component_state_scope_session", "model_registry_instance")
+@pytest.mark.usefixtures("updated_dsc_component_state_scope_session")
```

Comment:
here you want to remove `updated_dsc_component_state_scope_session` and keep `model_registry_instance`